.gitignore

```diff
@@ -3,4 +3,5 @@
 .settings/
 .theia/
 .vscode/
-target/
\ No newline at end of file
+target/
+/home/smwapinza/software-product-sprint/team8-project/src/main/webapp/client_secret_424069108958-6vthh91e0fds1gf6b0lbg3ld75jocohu.apps.googleusercontent.com.json
```

Comment:
Since the file name is the client secret, putting this in the gitignore still exposes the secret.. 

If we need the secret, I'd recommend just creating a separate file called "client_secret" or something and putting it in there so the contents of the file aren't shared. Otherwise, if the file name needs to be the secret, then just don't add it to git when you commit.

But other question is, do we need this? I don't see a reference to the client secret any where in the code